planning/route_handler/src/route_handler.cpp

```diff
@@ -793,9 +793,16 @@ bool RouteHandler::getNextLaneletWithinRoute(
   if (exists(goal_lanelets_, lanelet)) {
     return false;
   }
+
+  lanelet::ConstLanelet start_lanelet;
+  bool flag_check = true;
+  if (!getClosestLaneletWithinRoute(route_ptr_->start_pose, &start_lanelet)) {
+    flag_check = false;
+  }
```

Comment:
```suggestion
  const bool flag_check = getClosestLaneletWithinRoute(route_ptr_->start_pose, &start_lanelet);
```
@shulanbushangshu 
Sorry for the delay.
This part can be simplified.

I am currently running degradation test internally. If there is no issue, I will approve the PR